.github/workflows/build.yaml

```diff
@@ -70,19 +70,34 @@ jobs:
           MACOSX_DEPLOYMENT_TARGET: 10.14
         run: |
           cargo build --verbose --release
-          mv target/release/eslauncher2 eslauncher2-x86_64-apple-darwin
+          chmod 755 target/release/eslauncher2
+          ls -al target/release
+      - name: Package binary
+        run: |
+          cargo install cargo-bundle
+          cargo bundle --release
+          ls -al target/release/bundle/osx/ESLauncher2.app/Contents/MacOS
       - name: Strip binary
-        run: strip eslauncher2-x86_64-apple-darwin
-      - name: Archive binary
-        uses: actions/upload-artifact@v1
+        run: |
+          strip target/release/bundle/osx/ESLauncher2.app/Contents/MacOS/eslauncher2
+          chmod 755 target/release/bundle/osx/ESLauncher2.app/Contents/MacOS/eslauncher2
+          ls -al target/release/bundle/osx/ESLauncher2.app/Contents/MacOS
```

Comment:
We can leave this `ls` in just to be sure, but the step should be renamed.

Reply:
Will be renamed